semantic/src/definitions/modules/dimmer.js

```diff
@@ -0,0 +1,733 @@
+/*!
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected space or tab after '/*' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)